lib/ios/RCTHelpers.m

```diff
@@ -127,7 +127,13 @@ + (NSMutableDictionary *)textAttributesFromDictionary:(NSDictionary *)dictionary
 
 + (NSMutableDictionary *)textAttributesFromDictionary:(NSDictionary *)dictionary withPrefix:(NSString *)prefix
 {
-	return [self textAttributesFromDictionary:dictionary withPrefix:prefix baseFont:[UIFont systemFontOfSize:[UIFont systemFontSize]]];
+#if !(TARGET_OS_TV)
+	CGFloat systemFontSize = [UIFont systemFontSize];
+#else
+	CGFloat systemFontSize = 20;
```

Comment:
Worth pointing out I've plucked this default value out of thin air; tvOS doesn't have a `systemFontSize`.